core/src/specialized_solver/solid_mechanics/hyperelasticity/01_material_computations_elasticity_tensor.tpp

```diff
@@ -408,17 +408,6 @@ void HyperelasticityMaterialComputations<Term, withLargeOutput, MeshType,
 
   // output for debugging
   if (false) {
-    LOG(DEBUG) << "elasticity tensor, Ψ: "
-               << Term::strainEnergyDensityFunctionIsochoric;
-    LOG(DEBUG) << "∂Ψ/∂Ibar1: " << dPsi_dIbar1Expression;
-    LOG(DEBUG) << "∂Ψ/∂Ibar2: " << dPsi_dIbar2Expression << " = "
-               << dPsi_dIbar2;
-    LOG(DEBUG) << "∂2Ψ/(∂Ibar1 ∂Ibar1): " << d2Psi_dIbar1Ibar1Expression
-               << " = " << d2Psi_dIbar1Ibar1;
-    LOG(DEBUG) << "∂2Ψ/(∂Ibar1 ∂Ibar1): " << d2Psi_dIbar1Ibar2Expression
-               << " = " << d2Psi_dIbar1Ibar2;
-    LOG(DEBUG) << "∂2Ψ/(∂Ibar1 ∂Ibar1): " << d2Psi_dIbar2Ibar2Expression
-               << " = " << d2Psi_dIbar2Ibar2;
```

Comment:
What is the type of these variables? I cannot easily conclude that, but I understand they should be at least convertible to `double`. Would a `static_cast<double>(...)` work?

Reply:
Unfortunately this doesn't work:
> error: invalid ‘static_cast’ from type ‘SEMT::Expr<SEMT::Plus<SEMT::Plus<SEMT::Times<SEMT::Integer<0>, SEMT::Minus<SEMT::Variable<0>, SEMT::Integer<3> > >, SEMT::Times<SEMT::Parameter<0>, SEMT::Minus<SEMT::Integer<1>, SEMT::Integer<0> > > >, SEMT::Integer<0> > >’ to type ‘double’